Sensus.Android/Callbacks/AndroidNotifier.cs

```diff
@@ -55,7 +56,14 @@ public override void IssueNotificationAsync(string title, string message, string
                         .SetAutoCancel(true)
                         .SetOngoing(false);
 
-                    if (playSound)
+                    // if a protocol ID has been passed to the method, check the protocol's Notification Alert Exclusion Windows
+                    // to determine whether to vibrate and play sound. if protocolId is null, just use the vibrateAndPlaySound parameter.
+                    if (protocolId != null)
+                    {
+                        vibrateAndPlaySound = !NotificationTimeIsWithinAlertExclusionWindow(protocolId, DateTime.Now);
```

Comment:
If the `vibrateAndPlaySound` passed into this method is `false` but the current time is outside of the exclusion windows (or there are no exclusion windows), then sound and vibration will be used. Right? If so, this is not what the caller will expect. If the method parameter is false, then sound/vibration should not be used. If it's true, then sound/vibration should only be used if the exclusion windows allow (or if there are none).